### PR TITLE
swagger driverssummaryresponse: add activeMs, onDutyMs, driveMs, remo…

### PR DESCRIPTION
**Pull request by sskhandek**
…ve activeHours

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -3194,17 +3194,29 @@
...
+                    "description": "Milliseconds spent on duty or driving",
                     "example": 5.40
                   },
-                  "distanceMiles": {
+                  "onDutyMs": {
                     "type": "number",
                     "format": "float",
-                    "description": "Distance driven in miles, rounded to two decimal places.",
-                    "example": 123.24
+                    "description": "Milliseconds spent on duty",
```
Comment (silverspace) [inline swagger.json]: Nit: `Duration in milliseconds then driver was ON DUTY during the requested time range.`

File: swagger.json [hunk 2]
```diff
@@ -3194,17 +3194,29 @@
...
+                    "example": 5.40
```
Comment (silverspace) [inline swagger.json]: This should not be a floating point number.

File: swagger.json [hunk 3]
```diff
@@ -3194,17 +3194,29 @@
...
+                  "activeMs": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "Milliseconds spent on duty or driving",
                     "example": 5.40
                   },
-                  "distanceMiles": {
+                  "onDutyMs": {
                     "type": "number",
                     "format": "float",
```
Comment (silverspace) [inline swagger.json]: The number format should be int64, not float.

**Comment by sskhandek** [comment]
@silverspace Yup, it renders correctly

File: swagger.json [hunk 4]
```diff
@@ -3194,17 +3194,29 @@
...
+                    "type": "number",
+                    "format": "int64",
+                    "description": "Duration in milliseconds that driver was on duty or driving during the requested time range",
+                    "example": 12345
+                  },
+                  "onDutyMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "Duration in milliseconds that driver was on duty during the requested time range",
+                    "example": 12345
```
Comment (silverspace) [inline swagger.json]: Nit: Could we use a more realistic duration example?

Comment (sskhandek) [reply swagger.json]: Copied over values such as `43200000` from `fleet_test.json`

**Pull request merged**